packages/site/cms-site/src/blocks/helpers/VideoPreviewImage.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import styled from "styled-components";
+
+import { PixelImageBlockData } from "../../blocks.generated";
+import { PixelImageBlock } from "../PixelImageBlock";
+
+export interface VideoPreviewImageProps {
+    onClick: () => void;
+    image: PixelImageBlockData;
+    aspectRatio?: string;
+    sizes?: string;
+}
+
+export const VideoPreviewImage = ({ onClick, image, aspectRatio, sizes = "100vw" }: VideoPreviewImageProps) => {
+    const intl = useIntl();
+    return (
+        <Root onClick={onClick} aria-label={intl.formatMessage({ id: "videoPreviewImage.ariaLabel.startVideo", defaultMessage: "Start video" })}>
+            <PixelImageBlock
+                data={image}
+                aspectRatio={aspectRatio ? aspectRatio : "16x9"}
+                layout={aspectRatio ? "responsive" : "intrinsic"}
```

Comment:
This should probably not be done with layout since it's deprecated

Reply:
at this state it is required, when #2245 is merged, we must update this.

Reply:
Could you add a TODO?

Reply:
PR has been merged, please update and remove the prop.

Reply:
done 👍